.gitignore

```diff
@@ -1,2 +1,3 @@
 /bazel-*
 profile.out
+.idea/
```

Comment:
This is not an appropriate place for individual-user-specific project files; use .git/info/exclude instead. See  https://github.com/golang/go/issues/21458 for more details.